lab_1_keywords_tfidf/main.py

```diff
@@ -2,7 +2,29 @@
 Lab 1
 Extract keywords based on frequency related metrics
 """
-from typing import Optional, Union
+from typing import Optional, Union, Any
+from string import punctuation
+from math import log
+
+
+def check(obj: Any, exp_type: Any, exp_cont: Any = None, exp_val: Any = None, not_empty: bool = False) -> bool:
```

Comment:
bad name for this function. Check what? Also - bad names of agruments

Reply:
Fixed